lint

```diff
@@ -145,6 +145,18 @@ lint_sh() {
     return $lint_result
 }
 
+lint_tf() {
+    filename="$1"
```

Comment:
I realise this is copy-pasted from other functions, and happens to not do anything nasty right now, but shouldn't this be `local filename` to avoid overwriting the filename that `lint()` is using?